domain/src/main/java/org/oppia/domain/question/QuestionTrainingController.kt

```diff
@@ -3,23 +3,21 @@ package org.oppia.domain.question
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import org.oppia.app.model.Question
-import org.oppia.domain.topic.TEST_SKILL_ID_0
-import org.oppia.domain.topic.TEST_SKILL_ID_1
-import org.oppia.domain.topic.TEST_SKILL_ID_2
+import org.oppia.domain.topic.TopicController
 import org.oppia.util.data.AsyncResult
+import org.oppia.util.data.DataProvider
 import org.oppia.util.data.DataProviders
 import javax.inject.Inject
 import javax.inject.Singleton
 
-private const val QUESTION_DATA_PROVIDER_ID = "QuestionDataProvider"
-const val TEST_QUESTION_ID_0 = "question_id_0"
-const val TEST_QUESTION_ID_1 = "question_id_1"
-const val TEST_QUESTION_ID_2 = "question_id_2"
+const val TOTAL_QUESTIONS_PER_TOPIC = 10
```

Comment:
Is this constant name correct? Is this supposed to be QUESTION_COUNT_PER_TRAINING_SESSION? Won't there generally be more than 10 questions per topic?

Reply:
Done